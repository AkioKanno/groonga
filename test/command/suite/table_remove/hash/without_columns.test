table_create Names TABLE_HASH_KEY ShortText

dump

table_remove Names

dump
