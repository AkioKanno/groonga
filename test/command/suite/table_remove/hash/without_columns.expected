table_create Names TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
dump
table_create Names TABLE_HASH_KEY ShortText
table_remove Names
[[0,0.0,0.0],true]
dump
